coronavirus.md

```diff
@@ -62,13 +62,19 @@ Coming soon ...
 
 ### Italy
 
-### United Kingdom
+### United Kingdom
 
 ### France
 
 
 ## Models
-
-This section will list relevant modelling efforts.
-
-
+There are various types of models in epidemiology, which are used for prediction of prevalence (total number of infected) or the duration of an epidemic spreading processes. One of the most commonly used types is compartmental models, where population is divided into compartments of people with the same properties, e.g. people who are susceptible to virus, who are recovered etc.
+There are three main types of compartmental models in epidemiology: 
+    Susceptible-Infected (SI), where each person can be just in two states, either susceptible (S) or infected (I); 
+     Susceptible-Infected-Recovered (SIS), where each person can be in one of two states, susceptible (S) or infected (I), but then can become susceptible again;
+     Susceptible-Infected-Recovered (SIR), where each person can be  in one of three states, susceptible (S), infected (I) or recovered (R).
+     
+In SIR model we study the number of people in each of the compartments, denoted by variable S, I and R correspondingly. SIR system can be  expressed by the set of ordinary differential equations proposed by [O. Kermack and Anderson Gray McKendrick](https://en.wikipedia.org/wiki/Compartmental_models_in_epidemiology). 
+The simulations of SIR model are shown in the python script [here](https://github.com/Liyubov/heterogeneous-dynamics-on-networks/blob/master/code_network_heterogen_models/spreading_SIR.py).
+
+The main take-away message from the SIR model simulations is that we observe so-called peak of epidemic, when the number of infected people reaches its maximum. This moment of when size of epidemic outbreak is maximal depends on SIR model parameters: probability of epidemic transmission and recovery from the disease. 
```

Comment:
English grammar here e.g. "This moment of when size of epidemic" (should be "The timing of this peak moment, when the size of the epidemic is maximal, depends on SIR model ..."

And could you say more about that ...